canvas_modules/harness/cypress/support/canvas/verification-cmds.js

```diff
@@ -86,6 +109,19 @@ Cypress.Commands.add("verifyCommentIsNotSelected", (commentText) => {
 	cy.isCommentSelected(commentText).should("eq", false);
 });
 
+Cypress.Commands.add("verifyCommentIsAdded", (commentText) => {
```

Comment:
Let's change this to `verifyCommentExists`  because it might be used to check if, say, comments exist in a canvas that was loaded. Also change the corresponding method name for nodes and links if they exist.